package.json

```diff
@@ -67,14 +67,16 @@
     "postcss-loader": "4.2.0",
     "progress-stream": "2.0.0",
     "qrcode.react": "1.0.0",
+    "rc-rate": "2.9.1",
```

Comment:
Do we need to use this package? I don't see it used in the code. If we don't need to use it. Please remove